packages/components/src/dropdown-menu/dropdown-menu.tsx

```diff
@@ -123,7 +123,7 @@ Search.displayName = 'Search'
 type ItemProps = DropdownMenu.DropdownMenuItemProps & {
   icon?: IconElement
   label: string
-  onSelect: () => void
+  // onSelect: NonNullable<DropdownMenu.DropdownMenuItemProps['onSelect']>
```

Comment:
not sure if make it required or not :)

Reply:
> not sure if make it required or not :)

is it required by dropdown menu? (doesn't seem so, I'd follow it)

Reply:
That's true. I did, however, made it required here. 

 https://github.com/prichodko/status-web/blob/89ee596cf9b5d949cf184bbc132190c2269089b1/packages/components/src/dropdown-menu/dropdown-menu.tsx#L192-L193